wallaby.conf.js

```diff
@@ -0,0 +1,15 @@
+module.exports = function(wallaby) {
```

Comment:
try out wallaby if you haven't, 15 days trial, first year is a bit expensive, about 3k thb, second year is 2k, then from 3 years a bit more cheap (can't remember how much), I'll be renewing my license day after tomorrow (second year).